scripts/vendors/microsoft/setup_environment.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+import tarfile
+import zipfile
+import os
+from pathlib import Path
+from platform import system
+
+def get_test_resources_path(destination) -> str:
+    child_directories = [child for child in destination.iterdir() if child.is_dir()]
+    test_jdk_home = None
+    testimage_path = None
+
+    # should only be two directories to check for
```

Comment:
What if there are 3? What if there are 1?

Reply:
This is a directory which is created when we extract the contents of the compressed files. It is expected that we only have the jdk and its testimage in that directory.